.github/workflows/security.yml

```diff
@@ -80,38 +77,41 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: cloudposse/github-action-matrix-outputs-read@e0de318159d7139fb9e5f1d59e7f43f91d1d0904 # v0.1.2
-        id: trivy
+      - uses: actions/download-artifact@v4
         with:
-          matrix-step-name: trivy
+          pattern: trivy-results-*
+          path: results
+
+      - name: Merge results
+        run: |
+          find results -maxdepth 2 -name results.json -exec cat {} \; | jq -c --slurp 'map(to_entries | map(.key as $matrix_key | .value | map_values({($matrix_key): .}))) | flatten | reduce .[] as $item ({}; . * $item)' | tee results.json
 
       - name: Create GitHub issues
         run: |
-          echo "$result" | jq -c '[.vulnerabilities | to_entries[] | .key as $project | .value // empty | map(. + {Projects: [$project]})] 
+          jq -c 'to_entries | map((.value // empty) + {Projects: [.key]})
             | flatten 
             | group_by(.VulnerabilityID) 
             | map(reduce .[] as $vuln (.[0] + {Locations:[]}; .Projects += $vuln.Projects | .Locations += [$vuln.PkgName + ":" + $vuln.InstalledVersion + " (" + $vuln.PkgPath + ")"]))
             | map_values({Title: .VulnerabilityID, Body: ("### " + .Title + "\n" + .PrimaryURL + "\n>" + .Description + "\n#### Projects:\n* " + (.Projects | sort | unique | join("\n* ")) + "\n#### Locations:\n* `" + (.Locations | sort | unique | join("`\n* `")) + "`\n#### References:\n* " + (.References | sort | unique | join("\n* ")))})
-            | .[]' \
+            | .[]' < results.json \
           | while read -r vulnerability; do
-            title=$(echo "$vulnerability" | jq -r '.Title')
-            body=$(echo "$vulnerability" | jq -r '.Body')
+            title=$(jq -r '.Title' <<< "$vulnerability")
+            jq -r '.Body' <<< "$vulnerability" | tee "${title}-body.json"
             existing=$(gh issue list --state open --label dependencies --label security --search "$title" --json 'number' --jq '.[].number' | head -n1)
             if [ -n "$existing" ]; then
               echo "Issue '$title' already exists, updating body..."
-              gh issue edit "$existing" --body "$body"
+              echo gh issue edit "$existing" --body-file "${title}-body.json"
```

Comment:
Is the `echo` supposed to be in here?

Reply:
Oops - good spot, accidentally added that back in when fixing the linting issues.